### PR TITLE
[release-v1.14] Add func v1.15

### PR DESCRIPTION
**Pull request by dsimansk**

### CODE REVIEW

File: openshift/release/kn.yaml [hunk 1]
```diff
@@ -11,12 +11,11 @@ plugins:
...
-  #   replace:
-  #     - module: knative.dev/func
-  #       moduleSource: github.com/openshift-knative/kn-plugin-func
-  #       version: serverless-1.34
-  
\ No newline at end of file
+  - name: kn-plugin-func
+    module: knative.dev/func
+    pluginImportPath: knative.dev/func/plugin
+    version: v1.15.0
```
Comment (matejvasek) [inline openshift/release/kn.yaml:17]: Shouldn't this be v1.15.1?

Comment (dsimansk) [reply openshift/release/kn.yaml:17]: If we want to be very precise. :)

Thanks, good catch!

File: hack/build.sh.d/func.sh [hunk 2]
```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+# There no midstream image currently, fallback to upstream one.
+readonly func_util_image="${KN_PLUGIN_FUNC_UTIL_IMAGE:-quay.io/boson/alpine-socat:1.7.4.3-r1-non-root}"
```
Comment (matejvasek) [inline hack/build.sh.d/func.sh:4]: nit: the default should be `ghcr.io/knative/func-utils:latest`

Comment (matejvasek) [inline hack/build.sh.d/func.sh:4]: nit: the default here should be ghcr.io/knative/func-utils:latest

**Pull request merged**